packages/generator-particle-drupal/generators/module/index.js

```diff
@@ -0,0 +1,128 @@
+const Generator = require('yeoman-generator');
+
+// Module Dependencies
+const chalk = require('chalk');
+const {
+  camelCase,
+  kebabCase,
+  snakeCase,
+  startCase,
+  upperFirst,
+} = require('lodash');
+const path = require('path');
+const fs = require('fs');
+const rename = require('gulp-rename');
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+  }
```

Comment:
Again noting that this constructor is unnecessary unless we are handling args or opts.

Reply:
Ignore me. This may work without a constructor but the docs do refer to it being important to have in the code.